src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/sge/SGEJobManager.groovy

```diff
@@ -28,35 +28,9 @@ class SGEJobManager extends PBSJobManager {
         super(executionService, parms)
```

Comment:
Did you already merge the latest develop version?

Reply:
Yes